### PR TITLE
Add ps command to list stack-based containers currently running

### PR DESCRIPTION
**Pull request by henrynash**
Provide the option (along the lines of docker ps) to list details of the stack-based containers that are currently running in the docker environment.

Fixes: https://github.com/appsody/appsody/issues/69

### CODE REVIEW

**Comment by henrynash** [comment]
@tomleah Good feedback...just pushed changes you requested

**Comment by tomleah** [comment]
Thanks! LGTM 🙂 Would you also be able to create tests for this @henrynash

**Comment by henrynash** [comment]
@tomleah working on that now....

File: functest/ps_test.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	}
+
+	// now run appsody ps and see if we can spot the container
+	fmt.Println("about to run appsody ps")
+	stopOutput, errStop := cmdtest.RunAppsodyCmd([]string{"ps"}, projectDir)
+	if !strings.Contains(stopOutput, "CONTAINER") {
+		t.Fatal("output doesn't contain header line")
+	}
+	if !strings.Contains(stopOutput, containerName) {
+		t.Fatal("output doesn't contain correct conatiner name")
```
Comment (tomleah) [inline functest/ps_test.go]: container*

File: functest/ps_test.go [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+			time.Sleep(2 * time.Second)
+			count = count - 1
+		}
+		if count == 0 || containerRunning {
+			break
+		}
+	}
+
+	if !containerRunning {
+		t.Fatal("container never appeared t ostart")
```
[the remark this answers is not in the document]
Comment (henrynash) [reply functest/ps_test.go]: done

File: functest/ps_test.go (hunk 1 above)
Comment (henrynash) [reply functest/ps_test.go]: done

**Comment by kewegner** [comment]
I think it looks good lets commit this and merge it in

**Comment by henrynash** [comment]
@tomleah could you just confirm you are happy with this now?  I think I addressed all your comments (..but can't see how to mark the changes requested as "done"). If so, are you OK merging it?

**Comment by tomleah** [comment]
Thanks for this contribution @henrynash!

**Pull request merged**